scripts/winesapos-install.sh

```diff
@@ -570,7 +572,7 @@ if [[ "${WINESAPOS_INSTALL_PRODUCTIVITY_TOOLS}" == "true" ]]; then
     echo "Installing additional packages from the AUR complete."
 
 else
-    pacman_install_chroot lm_sensors man-db nano openssh rsync sudo terminator tmate tmux wget vim zstd
+    pacman_install_chroot lm_sensors man-db nano openssh rsync sudo terminator tmate tmux wget vi vim zstd
```

Comment:
Nice, thank you!

`vi` also needs to be added/installed about 20 lines up right after the `echo "Installing additional packages..."` line. Basically, the top section of the `if` block is for the performance and secure images and this line here is for the minimal image.